server/docker-entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+if type nvidia-smi > /dev/null;
```

Comment:
Can you change this to
```
if type nvidia-smi >/dev/null 2>/dev/null;
```
to also suppress the `type: nvidia-smi: not found` error message?